Interview_Prep_App/native_java/retrofit/src/main/res/layout/activity_anim_splash_screen.xml

```diff
@@ -4,31 +4,31 @@
     xmlns:tools="http://schemas.android.com/tools"
```

Comment:
Please restore the original file here and create a new xml to link yours to. I'd like to save other's work in the project